science/nektarpp/Portfile

```diff
@@ -0,0 +1,127 @@
+
+PortSystem             1.0
+PortGroup              cmake 1.1
+PortGroup              mpi 1.0
+PortGroup              gitlab 1.0
+
+name                   nektarpp
+
+gitlab.instance        https://gitlab.nektar.info
+gitlab.setup           nektar nektar 5.0.0 v
+
+categories             science
+platforms              darwin
+license                MIT
+homepage               http://www.nektar.info/
+maintainers            {xyloid.org:dave @mdave} openmaintainer
+description            spectral/hp element framework
+long_description       Nektar++ is a spectral/hp element framework which aims to be easy \
+                       to use and support a wide range of features, including 1D, 2D and \
+                       3D element support, hybrid shaped elements and continuous and     \
+                       discontinuous implementations. As well as a modern C++ API, it    \
+                       ships with a range of solvers including CFD, advection-diffusion  \
+                       equations and cardiac electrophysiology.
+
+checksums              rmd160  f588a4d6429b1ffc85be538885cbb09886ac9cba \
+                       sha256  d98c87af64e6d5cce04f8f60045a63fe63a79274c72a0ba368716fe603a4b625 \
+                       size    48486184
+
+compiler.cxx_standard  2011
+cmake.build_type       Release
+
+mpi.setup
+
+depends_lib-append     port:arpack \
+                       port:boost \
+                       port:fftw-3 \
+                       port:opencascade \
+                       port:scotch \
+                       port:tinyxml \
+                       port:zlib
+
+# Don't build demos and tests by default.
+configure.args-append \
+    -DNEKTAR_BUILD_DEMOS=OFF           \
+    -DNEKTAR_BUILD_TESTS=OFF           \
+    -DNEKTAR_BUILD_UNIT_TESTS=OFF      \
+    -DNEKTAR_SOLVER_DIFFUSION=OFF      \
+    -DNEKTAR_SOLVER_DUMMY=OFF          \
+    -DNEKTAR_SOLVER_DIFFUSION=OFF      \
+    -DNEKTAR_SOLVER_ELASTICITY=OFF     \
+    -DNEKTAR_SOLVER_DIFFUSION=OFF      \
+    -DNEKTAR_SOLVER_MMF=OFF            \
+    -DNEKTAR_USE_ARPACK=ON             \
+    -DNEKTAR_USE_FFTW=ON               \
+    -DNEKTAR_USE_SCOTCH=ON             \
+    -DNEKTAR_USE_MESHGEN=ON
+
+variant demos description {
+    Build additional Nektar++ demos
+} {
+    configure.args-append -DNEKTAR_BUILD_DEMOS=ON
+}
+
+variant petsc description {
+    Enable PETSc support for linear algebra solvers.
+} {
+    configure.args-append -DNEKTAR_USE_PETSC=ON
+    depends_lib-append port:petsc
+}
+
+if {[ mpi_variant_isset ]} {
+    depends_lib-append port:hdf5
+    require_active_variants hdf5 [ mpi_variant_name ]
+    require_active_variants scotch [ mpi_variant_name ]
+    configure.args-append -DNEKTAR_USE_MPI=ON -DNEKTAR_USE_HDF5=ON
+}
+
+variant oce description {
+    use Community Edition version of OpenCASCADE
+} {
+    depends_lib-replace port:opencascade port:oce
+}
+
+set pythons_suffixes    {27 36 37 38}
```

Comment:
Unless you have a specific reason to keep the older pythons, only use python 37 and 38 here.